test/app/reducers/misc.test.js

```diff
@@ -70,9 +70,9 @@ describe('misc reducers', () => {
       });
       let expectedResult = _.pickBy(devices, filterDevicesFn('win'));
       expect(actualResult).to.deep.equal(expectedResult);
-      // at least one device is unavailable on Windows, so available devices should be less than
+      // at least one device may be unavailable on Windows, so available devices should be less than
```

Comment:
The comment should also be changed to reflect the condition change to "at most".